content/docs/gto/user-guide/index.md

```diff
@@ -79,7 +113,7 @@ Alternatively, you can use environment variables (note the `GTO_` prefix)
 $ GTO_EMOJIS=false gto show
 ```
 
-## Git tag message format
+## Git tags format
```

Comment:
No, it's Git tag name. E.g. to register a version, you need a Git tag called `model@v0.0.1`, annotated with some text. It could be annotated with any text, but without a text annotation it would be a `lightweight` tag, not an `annotated` one. For example, see https://github.com/iterative/example-gto/tags

<img width="342" alt="image" src="https://user-images.githubusercontent.com/6797716/204767470-3e7a0dc2-5827-426d-806b-333be38365b3.png">

`churn@v3.1.1` is a tag name and `Registering artifact churn version v3.1.1` is a Git tag message.

Reply:
> It could be annotated with any text, but without a text annotation it would be a lightweight tag... 

I see. May I ask why does GTO ignore lightweight tags?

Reply:
Sure, we had a discussion about it here https://github.com/iterative/gto/issues/127